### PR TITLE
Load jQuery and Bootstrap from cdnjs

### PR DESCRIPTION
**Pull request by ricordisamoa**
Lots of libraries are now mirrored on Tool Labs:
https://lists.wikimedia.org/pipermail/labs-l/2015-April/003670.html

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
     <title>Contact info</title>
 
-    <link rel="stylesheet" href="//tools-static.wmflabs.org/static/bootstrap/3.1.1/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.1.1/css/bootstrap.min.css" />
     <link rel="stylesheet" href="//tools-static.wmflabs.org/static/bootstrap/tool/tools-footer.css" />
```
Comment (ricordisamoa) [inline templates/base.html:11]: 404 Not Found

**Comment by valhallasw** [comment]
//tools-static.wmflabs.org/static/bootstrap/3.1.1/css/bootstrap.min.css works for me?

**Comment by ricordisamoa** [comment]
Yes, my comment was about //tools-static.wmflabs.org/static/bootstrap/tool/tools-footer.css

**Comment by valhallasw** [comment]
Sorry, I never got back to you on this. bd808 integrated this with some changes, so the changes are (finally) deployed. Again, sorry for the slow feedback.

**Comment by ricordisamoa** [comment]
Never mind :smile:

**Pull request closed without merging**